sdk/cdn/arm-cdn/_meta.json

```diff
@@ -1,8 +1,8 @@
 {
-  "commit": "82fdf284eea880a8e027c354911a814999a77178",
```

Comment:
@dw511214992 We can't find the commit id do you know why this happens?

Reply:
This is a release not refresh, we need to take latest swagger as input.

Reply:
If the commit id is in local, and not push to spec repo, you cannot find it.